src/api-calls/addToCart.js

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+import { cartActionsConstants } from "../reducer/index";
+
```

Comment:
I am sure you will remove these extra spaces where its not  necessary later while refactoring your code.